src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -64,34 +70,44 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1 2345")]
         [InlineData("HTTP/A.1 200 OK")]
         [InlineData("HTTP/X.Y.Z 200 OK")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on .NET Framework.
-        //[InlineData("HTTP/0.1 200 OK")]
-        //[InlineData("HTTP/3.5 200 OK")]
```

Comment:
You deleted the 2 test cases above. Any reason why? Per original Stephen's comment they were passing on .NET Core on Windows. Does it mean we do not throw the exception anymore on .NET Core?

Reply:
Yes, these two test data will not throw exception anymore on .NET Core.

Reply:
I wonder what they return and if it is a regression we should address.

Reply:
On .NET Core 2.0, `[InlineData("HTTP/0.1 200 OK")]` `[InlineData("HTTP/3.5 200 OK")]` will result in `HttpRequestException`. But current behavior is not throwing Exception. (6 & 7 cases in my comment below)